event-gateway/WEBBROKERAPI.md

```
@@ -0,0 +1,1653 @@
+- [Specification Format](#specification-format)
+- [Example Use Cases](#example-use-cases)
+- [Building and Running](#building-and-running)
+  - [Option 1: Using Docker Compose (Recommended)](#option-1-using-docker-compose-recommended)
+  - [Option 2: Building from Source](#option-2-building-from-source)
+  - [Option 3: Development Mode with Live Reload](#option-3-development-mode-with-live-reload)
+- [Testing with Policies](#testing-with-policies)
+- [End-to-End Testing with Kafka](#end-to-end-testing-with-kafka)
+- [Implementation Details](#implementation-details)
+- [Comparison: WebSubApi vs WebBrokerApi](#comparison-websubapi-vs-webbrokerapi)
+- [Consumer Group Strategy](#consumer-group-strategy)
+- [Topic Subscription](#topic-subscription)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)